tiny_mce/custom/langs/vi.js

```diff
@@ -0,0 +1,240 @@
+\ufeff/*
```

Comment:
@tarangkhandelwal 
No, it is not. Sorry for the trouble and thank you for noticing.
That is an UTF-8 BOM encoded in Unicode Escape and it shouldn't be there. Fixing it.